### ISSUE TITLE
Add to widgets by ID

### ISSUE THREAD

**Issue by hackmastera** [pulibrary/pomegranate#729]
The UI for adding an image to a widget is not ideal. For example, when selecting a page from a multi-page book all I see are small thumbnails. I can't always tell which thumbnail corresponds to the image I want, and I don't have any additional metadata.

Selecting an object itself does not give sufficient information if many objects (e.g. periodicals) have the same or similar titles.

There are other interfaces in Figgy and DPUL which are much easier to navigate.

As a curator I would like to specify an object or page by id so that I can work around the UI challenges of object / image selection in widgets.

Note: This use case may require more discussion: e.g. should we aim to enhance the widget UI instead of writing functionality to accept IDs?

**Comment by kelea99** [comment]
I agree, @hackmastera.  issue requires more discussion before prioritization.
